test/specs/behaviors/testHelper.tsx

```diff
@@ -60,25 +67,39 @@ export class TestHelper {
   public findRegexAndAssingCorrespondingInfoToArray(behaviorMenuItems: any) {
     behaviorMenuItems.forEach(behavior => {
       behavior.variations.forEach(variant => {
-        variant.text.split('\n').forEach(singleLineText => {
-          this.iterateRegexDefinitions(singleLineText, variant.name)
-        })
+        if (!variant.specification && !variant.text) {
+          this.createMissingDescriptionTest(variant.name)
+        }
+        if (!variant.specification) {
+          this.verifySpecificationTag(variant.name)
+        } else {
+          variant.specification.split('\n').forEach(singleLineText => {
+            this.iterateRegexDefinitions(singleLineText, variant.name)
+          })
+        }
       })
     })
   }
 
   public iterateRegexDefinitions(singleLineText: string, behaviorName: string) {
+    let regMatched = false
     this.testDefinitions.forEach(testDefinition => {
       const regex = new RegExp(testDefinition.regexp)
       const result = regex.exec(singleLineText)
       if (result) {
+        regMatched = true
         this.filteredDescriptionWithAssignedTestMethod.push({
           behaviorName,
           testMethod: testDefinition.testMethod,
           params: result,
         })
       }
     })
+    if (!regMatched) {
+      test(`${behaviorName} \n LINE: ${singleLineText} `, () => {
+        fail(`Line mentioned in test name doesn't match any regex expressions.`)
```

Comment:
I would suggest something that is more descriptive, like: `Line mentioned in **behavior specification** doesn't match any regex expression of validation tests.`